Personally created shield image

clapierre opened the pull request:
I just made this image of a shield with a checkmark creative commons Free to use :)

John246810 reviewed (approved):
Sisi

gregoriopellegrino commented:
I replaced the report icon with the Noun Project icon and cited the source in the Acknowledgements section.

Only the accessibility icon is missing: @clapierre do you have news about it? If not, my proposal is to replace it with another [Noun Project icon](https://thenounproject.com/search/?q=accessibility&i=1872), but it's in black and white.

clapierre commented:
Hello Gregorio and all
I couldn't find out any more information regarding the accessibility Icons at WAI<https://www.w3.org/WAI/teach-advocate/accessibility-icon-set> which are still it seems in draft status which was to be updated this past March.
I changed the single instance of the accessibility icon and added a section linking to the WAI icons page and followed their suggestion of  Creative Commons Attribution 4.0 International License, with attribution requirements which states to include the current status which is "draft".

I am sure Matt will be able to tweak our IP sections if needed.


Thanks
EOM
Charles LaPierre
Technical Lead, DIAGRAM and Born Accessible
Imageshare Product Manager
Twitter: @CLaPierreA11Y
Skype: charles_lapierre


On Aug 20, 2021, at 6:02 AM, Gregorio Pellegirno ***@***.******@***.***>> wrote:



I replaced the report icon with the Noun Project icon and cited the source in the Acknowledgements section.

Only the accessibility icon is missing: @clapierre<https://github.com/clapierre> do you have news about it? If not, my proposal is to replace it with another Noun Project icon<https://thenounproject.com/search/?q=accessibility&i=1872>, but it's in black and white.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/w3c/publ-a11y/pull/51#issuecomment-902676658>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AB3A3LOGFUEITO762Y57K3LT5ZG5PANCNFSM5BELKOXQ>.
